packages/core/capabilities/core-capabilities-server-internal/src/routes/resolve_capabilities.ts

```diff
@@ -39,7 +39,12 @@ export function registerCapabilitiesRoutes(router: IRouter, resolver: Capabiliti
     async (ctx, req, res) => {
       const { useDefaultCapabilities } = req.query;
       const { applications } = req.body;
-      const capabilities = await resolver(req, applications, useDefaultCapabilities);
+      const capabilities = await resolver({
+        request: req,
+        applications,
+        useDefaultCapabilities,
+        capabilityPath: ['*'],
```

Comment:
Looking at this particular instance of capabilityPath value I'm wondering if it is just simpler to set capabilityPath type as `string[]` instead of `string | string[]`.

Reply:
From a DX standpoint, it's not, most calls to the API are done with a single path. And it doesn't really add that much complexity to the implementation to justify impacting the DX unnecessarily.